class-gf-razorpay.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+require_once ('razorpay-sdk/Razorpay.php');
+use Razorpay\Api\Api;
+
+GFForms::include_payment_addon_framework();
+
+class GFRazorpay extends GFPaymentAddOn {
+
+    protected $_version = GF_RAZORPAY_VERSION;
+    protected $_min_gravityforms_version = '1.9.3';
+    protected $_slug = 'razorpay-gravity-forms';
+    protected $_path = 'razorpay-gravity-forms/razorpay.php';
+    protected $_full_path = __FILE__;
+    protected $_url = 'http://www.gravityforms.com';
+    protected $_title = 'Gravity Forms Razorpay Add-On';
+    protected $_short_title = 'Razorpay';
+    protected $_supports_callbacks = true;
+
+    // Permissions
+    protected $_capabilities_settings_page = 'gravityforms_razorpay';
+    protected $_capabilities_form_settings = 'gravityforms_razorpay';
+    protected $_capabilities_uninstall = 'gravityforms_razorpay_uninstall';
+
+    // Automatic upgrade enabled
+    protected $_enable_rg_autoupgrade = true;
+
+    private static $_instance = null;
+
+    public static function get_instance() {
```

Comment:
opening brace on the next line. Please fix this everywhere. There are many cases of this throughout the code base.